testnet/stacks-node/src/burnchains/bitcoin_regtest_controller.rs

```diff
@@ -757,7 +751,7 @@ impl BitcoinRegtestController {
         let btc_miner_fee = self.config.burnchain.block_commit_tx_estimated_size
             * self.config.burnchain.satoshis_per_byte;
 
-        let rbf_fee = (attempt.saturating_sub(1) * self.last_tx_len * self.min_relay_fee) / 1000;
+        let rbf_fee = attempt.saturating_sub(1) * btc_miner_fee;
```

Comment:
This is a pretty steep fee increase for rbf, right? This would lead to, for example, something that looked like:

```
tx_fee attempt 1: 40,000 satoshis
tx_fee attempt 2: 80,000 satoshis
tx_fee attempt 3: 120,000 satoshis
```

Would  a better increase be like `attempt.saturating_sub(1) * self.config.burnchain.block_commit_tx_estimated_size`?

Reply:
I think it is steep by design.  Consider rule 3 in BIP0125 [1], which defines RBF:

> The replacement transaction pays an absolute fee of at least the sum paid by the original transactions.

@lgalabru Can you test this out and verify that we cannot go lower than `attempt.saturating_sub(1) * btc_miner_fee`?

[1] https://github.com/bitcoin/bips/blob/master/bip-0125.mediawiki

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2363/commits/eaacaa7cdeef23f562c41ea0b4401baff8df90fe.
Will give it a try on Xenon.